src/parser/pdb-parser.ts

```diff
@@ -328,6 +330,12 @@ class PdbParser extends StructureParser {
             if (isPdbqt) {
               atomStore.partialCharge![ idx ] = parseFloat(line.substr(70, 6))
             }
+            if (isFinite(formalCharge)) {
```

Comment:
Yup, I'll add a comment. I think I'd err on the side of putting all the surprising code in the same place with a comment rather than also needing to comment NaN up top, unless you have a strong view otherwise?

`Number.isFinite()` isn't available in IE11, but the global `isFinite` is - the difference in behaviour is that `Number.isFinite('2.5')` is false, wherease `isFinite('2.5')` is true.  I don't think it makes a difference which we use here as it definitely won't be a string that gets passed in. I did a quick benchmark and perf on `NaN`, `undefined` and `1` is indistinguishable on Chrome at least.

Reply:
Good, thanks for the clarification!